test/content/ContentManagerTests.js

```diff
@@ -105,4 +124,53 @@ describe('Content Manager Contract Tests', () => {
             expect(fees.royaltyAmount).to.equal(10);
         });
     });
+
+    describe("Register Operator Tests", () => {
+        it('Same operator address', async () => {
+            var mintData = [playerAddress.address, [2], [5], 1, ethers.constants.AddressZero, []];
+
+            // craftingSystemAddress should have minter role revoked
+            var approvalPairs1 = [
+                [craftingSystemAddress.address, true], [craftingSystemAddress.address, true],
+                [craftingSystemAddress.address, false]
+            ];
+
+            await contentManager.registerOperators(approvalPairs1);
+            await expect(content.connect(craftingSystemAddress).mintBatch(mintData)).to.be.reverted;
+
+            expect(await content.totalSupply(2)).to.equal(0);
+
+            // craftingSystemAddress should have minter role granted
+            var approvalPairs2 = [
+                [craftingSystemAddress.address, false], [craftingSystemAddress.address, false],
+                [craftingSystemAddress.address, true]
+            ];
+
+            await contentManager.registerOperators(approvalPairs2);
+            await content.connect(craftingSystemAddress).mintBatch(mintData);
+            expect(await content.totalSupply(2)).to.equal(5);
+        });
+
+        it('Other parameter inputs', async () => {
+            var mintData = [playerAddress.address, [1], [100], 1, ethers.constants.AddressZero, []];
+
+            var approvalPairs1 = [[null, true]];
+            var approvalPairs2 = [["", false]];
+            var approvalPairs3 = [[ethers.constants.AddressZero, true], [craftingSystemAddress.address, true]];
+            // setting boolean to null would work as if it was set to false
+            var approvalPairs4 = [[deployerAddress.address, null]];
+
+            
+            await expect(contentManager.registerOperators(approvalPairs1)).to.be.reverted;
+            await expect(contentManager.registerOperators(approvalPairs2)).to.be.reverted;
+            await contentManager.registerOperators(approvalPairs3);
```

Comment:
I'm surprised that this is passing, that "AddressZero" doesn't revert or anything. Is that really the case?

Reply:
Actually, fix this. you didn't add "expect(function()).to.be.reverted" here;

Reply:
Ignore this. I was wrong here.